frontend/src/lib/k8s/resourceQuota.ts

```diff
@@ -42,35 +42,113 @@ class ResourceQuota extends makeKubeObject<KubeResourceQuota>('resourceQuota') {
 
   get requests(): string[] {
     const req: string[] = [];
-    Object.keys(this.spec.hard).forEach(key => {
-      if (key.startsWith('requests.')) {
-        req.push(`${key}: ${this.status.used[key]}/${this.spec.hard[key]}`);
-      }
-    });
+    this.spec.hard &&
+      Object.keys(this.spec.hard).forEach(key => {
+        if (key.startsWith('requests.')) {
+          req.push(
+            `${key}: ${this.normalizeUnit(key, this.status.used[key])}/${this.normalizeUnit(
+              key,
+              this.spec.hard[key]
+            )}`
+          );
+        }
+      });
```

Comment:
looks like this should also accept `cpu` and `memory` since it is same as `requests.cpu` and `requests.memory`.
[Refer](https://kubernetes.io/docs/concepts/policy/resource-quotas/#compute-resource-quota)